spec/terminology.md

```diff
@@ -3,17 +3,74 @@
 ::: todo
 :::
 
-[[def: method-specific identifier, MSI]]
-~ The `method-specific-id` part of DID Syntax, as defined in [DID Core](https://www.w3.org/TR/did-core/#did-syntax). See section [Method-Specific Identifier](#method-specific-identifier).
+[[def: authentic chained data container (ACDC), ACDC]]
+~ a variant of [the Verifiable Credential (VC) specification](https://www.w3.org/TR/vc-data-model/). Some of the major distinguishing features of ACDCs include normative support for chaining, use of composable JSON Schema, multiple serialization formats, support for Ricardian contracts, support for chain-link confidentiality compact formats for resource constrained applications, simple partial disclosure mechanisms and simple selective disclosure mechanisms. ACDCs inherit the security model derived from [[ref: KERI]].
+
+[[def: autonomic identifier (AID)]]
+~ A self-managing identifier that is self-certifying (i.e., self-authenticating) and cryptographically bound to a [[ref: key event log]] ([[ref: KEL]]). An AID is either non-transferable or transferable. A non-transferable AID does not support key rotation while a transferable AID support key rotation using a key [[ref: pre-rotation]] mechanism that enables the AID to persist in spite of the evolution of its key state.
+
+[[def: compact Event Streaming Representation (CESR), CESR]]
+~ A dual text-binary encoding format that has the unique property of text-binary concatenation composability. The composability property enables the round trip conversion en-masse of concatenated primitives between the text domain and binary domain while maintaining the separability of individual primitives.
+
+[[def: controller]]
+~ A controlling entity that can cryptographically prove the control authority (signing and rotation) over an [[ref: AID]], as well as make changes on the associated [[ref: KEL]]. A controller may consist of multiple controlling entities in a multi-signature scheme.
 
 [[def: DID document]]
 ~ As defined by [DID Core](https://www.w3.org/TR/did-core/#dfn-did-documents). See section [DID Documents](#did-documents).
 
+[[def: direct replay mode]]
+~ an operational mode of the [[ref: KERI]] protocol where a controller and a verifier of an [[ref: AID]] exchange the [[ref: KEL]] of the AID directly. Upon reception of the KEL, the verifier sends a key event receipt, which includes the verifier's signature, to the controller as an acknowledgment. The signed key event receipt attests that the verifier received and verified the key event message. 
+
+[[def: inception event]]
+~ A key establishment event that provides the incepting information needed to derive an [[ref: AID]] and establish its initial [[ref: key state]].
+
+[[def: indirect replay mode]]
+~ An operational mode of the [[ref: KERI]] protocol where the [[ref: KEL]] of an [[ref: AID]] is promulgated to a verifier using witnesses that are disignated by the AID's controller. 
```

Comment:
Relevant to did:webs?

Typo in “designated”.   And I’m not sure I would use the “promulgated”, but perhaps that’s just me.

Reply:
I think it is relevant. 

Promulgation is the term used by Sam in the whitepaper, but I agree it is not a commonly used term in English dictionaries. I will change it to "discovered by a verifier via witnesses".